misc/scripts/download.sh

```diff
@@ -31,8 +31,10 @@ if ! wget -q --tries=10 --timeout=20 --spider https://github.com; then
 fi
 
 if curl --output /dev/null --silent --head --fail "$URL" ; then
-	mkdir -p "$HOME/.cache/pacstall" && cd "$HOME"/.cache/pacstall/
-	mkdir -p "$PACKAGE" && cd "$PACKAGE"
+	if [[ "$type" = "install" ]]; then
+		mkdir -p "/tmp/pacstall/pacscripts/" && cd "/tmp/pacstall/pacscripts/"
+		mkdir -p "$PACKAGE" && cd "$PACKAGE"
```

Comment:
I don't think we need to use `mkdir -p "$PACKAGE" && cd "$PACKAGE"` anymore in install

Reply:
why?

Reply:
Because of how we source the pacscript, it doesn't matter where it is, only that we are in the right dir

Reply:
Alright, you know the best! On it.